pkg/libvmi/vmi.go

```diff
@@ -263,6 +263,12 @@ func WithoutSerialConsole() Option {
 	}
 }
 
+func WithStatus(status *v1.VirtualMachineInstanceStatus) Option {
```

Comment:
The package [rules](https://github.com/kubevirt/kubevirt/tree/main/pkg/libvmi#rules) requests not to add to this file things unless all other options have been exhausted.

Was there a discussion on this and reasoning about adding it here?

Furthermore, I think until now `libvmi` was focused on specifying the spec but did not touch the `status`. This by itself requires a discussion.

Reply:
@EdDev Same topic here: https://github.com/kubevirt/kubevirt/pull/12241

Reply:
@EdDev no it wasn't discussed, since the PR has already been merged, we can open a new one and remove it. Even if I personally don't find it weird of adding it.